qemu/tests/balloon_hotplug.py

```diff
@@ -51,6 +69,13 @@ def run(test, params, env):
                               logging.info)
         balloon_check.run(test, params, env)
 
+        if pm_test_after_plug:
+            run_pm_test(pm_test_after_plug, "hot-plug")
+            if vm.is_alive():
+                balloon_check.run(test, params, env)
+            else:
+                return
```

Comment:
Should use continue here, if pm_test_after_plug=shutdown, the function will quit, so only run 1 loop

Reply:
Probably just remove `else` part , if it won't do anything

Reply:
Yes,this is what I want.
If guest shut down,no need to execute followed tests.

Reply:
If remove else part,then followed unplug test will be executed,but actually guest is already shut down.
So I need to break it when pm_test is shutdown